tensorflow/tools/api/golden/v2/tensorflow.pbtxt

```diff
@@ -904,6 +904,10 @@ tf_module {
     name: "searchsorted"
     argspec: "args=[\'sorted_sequence\', \'values\', \'side\', \'out_type\', \'name\'], varargs=None, keywords=None, defaults=[\'left\', \"<dtype: \'int32\'>\", \'None\'], "
   }
+  member_method {
+    name: "select_v2"
```

Comment:
The selectv2 op needs an apidef file in the python_api directory to mark it as hidden